Makefile

```diff
@@ -8,7 +8,7 @@ BUILDING_SHARED=YES
 endif
 
 ## list all valid IOC directories that we may want to build at some point
-IOCDIRS = AG33220A AG3631A AG53220A CCD100 CONEXAGP CONTROLSVCS CRYVALVE DELFTARDUSTEP DELFTBPMAG DELFTDCMAG DELFTSHEAR DFKPS ECLAB EUROTHRM FINS GALIL HAMEG8123 HIFIMAG HVCAEN INSTETC ISISDAE JULABO KHLY2400 KEPCO LINKAM95 LKSH218 LKSH336 MCLEN MERCURY_ITC MK3CHOPR NANODAC NEOCERA PDR2000 PIMOT PSCTRL RUNCTRL SDTEST SKFCHOPPER SMC100 SPINFLIPPER306015 STPS350 STSR400 TDK_LAMBDA_GENESYS TEKAFG3XXX TEKDMM40X0 TEKMSO4104B TEST TPG26x TPG300 TTIEX355P SCIMAG3D
+IOCDIRS = AG33220A AG3631A AG53220A CCD100 CONEXAGP CONTROLSVCS CRYVALVE DELFTARDUSTEP DELFTBPMAG DELFTDCMAG DELFTSHEAR DFKPS ECLAB EUROTHRM FINS GALIL HAMEG8123 HIFIMAG HVCAEN INSTETC ISISDAE JULABO KHLY2400 KEPCO LINKAM95 LKSH218 LKSH336 MCLEN MERCURY_ITC MK3CHOPR MK2CHOPR NANODAC NEOCERA PDR2000 PIMOT PSCTRL RUNCTRL SCIMAG3D SDTEST SKFCHOPPER SMC100 SPINFLIPPER306015 STPS350 STSR400 TDK_LAMBDA_GENESYS TEKAFG3XXX TEKDMM40X0 TEKMSO4104B TEST TPG26x TPG300 TTIEX355P
```

Comment:
Nit-picking, but if we're doing alphabetical order then "MK2" should come before "MK3"